packages/recipe-ui-theme-marketplace/src/index.ts

```diff
@@ -7,16 +7,42 @@ const marketplace = createTheme({
   },
   borderWidths: {
     'card-border': '$thin',
+    'button-secondary': '$thin',
+    'button-secondary-focus': '$wide',
   },
```

Comment:
This difference in border-width is still causing a shift in the button for `focus`+`hover`. Instead, you could use `box-shadow` to create the border, like we do with the primary button.

```suggestion
  borderWidths: {
    'card-border': '$thin',
    'button-secondary': '$thin',
  },
```

Reply:
done!